WebDriverAgentLib/Commands/FBOrientationCommands.m

```diff
@@ -123,4 +124,20 @@ + (NSDictionary *)_orientationsMapping
   return orientationMap;
 }
 
++ (NSDictionary *)_wdOrientationsMapping
```

Comment:
underscore is not needed - the method is anyway not visible from the outside of the implementation block

Reply:
i was matching style with `_orientationsMapping`, but i can not